wireless_modules/wireless_module.py

```diff
@@ -14,16 +14,18 @@ class WirelessModule:
     """
     A class structure to read and collate data from different sensors into a dictionary and send through MQTT.
     """
-    def __init__(self, module_id):
+    def __init__(self, module_id, battery_reader=None):
         """
         Initialises the wireless module.
         :param module_id: An integer representing the wireless module number.
+        :param battery_reader: A `BatteryReader` class to read the battery voltage from
         """
         self.sensors = []
 
-        self.pub_data_topic = b"/v3/wireless_module/{}/data".format(module_id)
+        self.pub_sensor_topic = b"/v3/wireless_module/{}/data".format(module_id)
+        # FIXME: Need to verify the need for the low-battery topic
         self.pub_low_battery = b"/v3/wireless_module/{}/low-battery".format(module_id)
```

Comment:
Need to have a look at whether we need such a battery topic.

Reply:
I think we've decided that there's no need for the low battery topic.